Explorer/Assets/DCL/Minimap/MinimapController.cs

```diff
@@ -0,0 +1,141 @@
+            viewInstance.expandMinimapButton.onClick.AddListener(ExpandMinimap);
+            viewInstance.minimapRendererButton.onClick.RemoveAllListeners();
+            viewInstance.minimapRendererButton.onClick.AddListener(() => mvcManager.ShowAsync(ExplorePanelController.IssueCommand(new ExplorePanelParameter(null))).Forget());
+        }
+
+        private void ExpandMinimap() =>
+            viewInstance.minimapContainer.gameObject.SetActive(!viewInstance.minimapContainer.gameObject.activeSelf);
+
+        [All(typeof(PlayerComponent))]
+        [Query]
+        private void QueryPlayerPosition(in TransformComponent transformComponent)
+        {
+            var position = transformComponent.Transform.position;
+
+            if (mapCameraController == null)
+            {
+                mapCameraController = mapRenderer.RentCamera(new MapCameraInput(
+                    this,
+                    RENDER_LAYERS,
+                    Vector2Int.RoundToInt(MapRendererTrackPlayerPosition.GetPlayerCentricCoords(position)),
+                    1,
+                    viewInstance.pixelPerfectMapRendererTextureProvider.GetPixelPerfectTextureResolution(),
+                    new Vector2Int(viewInstance.mapRendererVisibleParcels, viewInstance.mapRendererVisibleParcels)
+                ));
+
+                mapRendererTrackPlayerPosition = new MapRendererTrackPlayerPosition(mapCameraController);
+                viewInstance.mapRendererTargetImage.texture = mapCameraController.GetRenderTexture();
+                viewInstance.pixelPerfectMapRendererTextureProvider.Activate(mapCameraController);
+                GetPlaceInfoAsync(position);
+            }
+            else
+            {
+                mapRendererTrackPlayerPosition.OnPlayerPositionChanged(position);
+                GetPlaceInfoAsync(position);
+            }
+        }
+
+        protected override void OnBlur()
+        {
+            mapCameraController.SuspendRendering();
+        }
+
+        protected override void OnFocus()
+        {
+            mapCameraController.ResumeRendering();
+
+            mapRenderer.SetSharedLayer(MapLayer.SatelliteAtlas, true);
+            mapRenderer.SetSharedLayer(MapLayer.ParcelsAtlas, false);
+        }
+
+        private void GetPlaceInfoAsync(Vector3 playerPosition)
+        {
+            Vector2Int playerParcelPosition = ParcelMathHelper.WorldToGridPosition(playerPosition);
+
+            if (previousParcelPosition == playerParcelPosition)
+                return;
+
+            previousParcelPosition = playerParcelPosition;
+
+            RetrieveParcelInfoAsync(playerParcelPosition).Forget();
+            return;
+
+            async UniTaskVoid RetrieveParcelInfoAsync(Vector2Int playerParcelPosition)
+            {
+                try
+                {
+                    PlacesData.PlaceInfo placeInfo = await placesAPIService.GetPlaceAsync(playerParcelPosition, CancellationToken.None);

```

Comment:
There should be a cancellation token, probably should incorporate one into `ControllerBase` that will be fired on `MVCManager` `Dispose`